core/src/main/scala/org/ensime/util/FileUtil.scala

```diff
@@ -84,14 +84,14 @@ object FileUtils {
     }
   }
 
-  def writeDiffChanges(changes: List[FileEdit], cs: Charset): Either[Exception, File] = {
+  def writeDiffChanges(changes: List[FileEdit])(implicit cs: Charset): Either[Exception, File] = {
     //TODO: add support for NewFile and DeleteFile
```

Comment:
Sry, I didn't have time last week to respond to this.

In here I meant that TODO entry however after studying our library refatoring possibilities I've come to conclusion that none of the refactorings will create/delete files.

As for implicit charset, it's passed explicitly in this case because of ambiguous implicit values from method parameter and https://github.com/ensime/ensime-server/blob/master/util/src/main/scala/org/ensime/util/file.scala#L31